src/offset_of.rs

```diff
@@ -200,6 +200,9 @@ macro_rules! offset_of_union {
 
 #[cfg(test)]
 mod tests {
+    #![cfg(allow_clippy)]
```

Comment:
This disables the entire test module when we dont allow_clippy. Don't do that please! We want the tests to run.

Reply:
Woops, I thought with it being `#!` that it wouldn't do that, but I see in the CI logs that it did. Pushed a version using `cfg_attr` that should be better ... hopefully.

Reply:
Seems better now!